calcom/cal.com
# Hotfix Settings Shell

## What does this PR do?



This PR moves settings navigation components to the shell component.

Fixes # (issue)

https://www.loom.com/share/ac988658393d425ea985d9ff3c8b4a69



**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Enable V2 and go to settings
- Change the window size and you should see the side container disappear and the top settings nav container appear
- Pressing on the hamburger menu should slide in the side container

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
